### PR TITLE
Mar 1041

### PR DESCRIPTION
**Pull request by jamespacileo**

### CODE REVIEW

File: config/settings/base.py [hunk 1]
```diff
@@ -247,7 +251,8 @@
     "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.LimitOffsetPagination",
     "PAGE_SIZE": 400,
     "DEFAULT_AUTHENTICATION_CLASSES": [
-        "oauth2_provider.contrib.rest_framework.OAuth2Authentication"
+        "oauth2_provider.contrib.rest_framework.OAuth2Authentication",
+        # "rest_framework.authentication.SessionAuthentication",
```
Comment (cad106uk) [inline config/settings/base.py:255]: Why is this commented out line here? Should we be using this authenticator or should we just delete this line?

**Review by cad106uk — CHANGES_REQUESTED** [review]
why does the empty file  api/action_plans/constants.py exist?

File: api/assessment/serializers.py [hunk 2]
```diff
@@ -219,8 +230,11 @@ class Meta:
...
             "reviewed_by",
             "reviewed_on",
-            "is_current"
+            "is_current",
         )
 
     def get_is_current(self, obj):
-        return EconomicAssessment.objects.filter(created_on__gt=obj.created_on).count() == 0
+        return (
+            EconomicAssessment.objects.filter(created_on__gt=obj.created_on).count()
```
Comment (cad106uk) [inline api/assessment/serializers.py:238]: Why not use the ORM method `exists()` for this?

**Review by cad106uk — CHANGES_REQUESTED** [review]
Looks good apart from the empty `api/action_plans/constants.py` file

File: api/assessment/serializers.py (hunk 2 above)
Comment (jamespacileostint) [reply api/assessment/serializers.py:238]: done!

File: api/action_plans/models.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    id = models.UUIDField(primary_key=True, default=uuid4)
+    action_plan = models.ForeignKey(
+        ActionPlan, related_name="milestones", on_delete=models.CASCADE
+    )
+
+    objective = models.TextField()
+    completion_date = models.DateField(null=True, blank=True)
+
+
+ACTION_PLAN_TASK_CHOICES = Choices(
```
[the remark this answers is not in the document]
Comment (jamespacileostint) [reply api/action_plans/models.py:52]: good plan! done!

File: api/assessment/serializers.py [hunk 4]
```diff
@@ -121,7 +128,10 @@ def create(self, validated_data):
         return super().create(validated_data)
 
     def get_is_current(self, obj):
-        return EconomicAssessment.objects.filter(created_on__gt=obj.created_on).count() == 0
+        return (
```
Comment (cad106uk) [inline api/assessment/serializers.py:131]: Should use the `exists()` method here

**Review by cad106uk — APPROVED** [review]

**Pull request merged**